kernelci/build.py

```diff
@@ -787,6 +807,17 @@ def install_kernel(kdir, tree_name, tree_url, git_branch, git_commit=None,
         shell_cmd("tar -C{path} -cJf {tarball} .".format(
             path=mod_path, tarball=modules_tarball_path))
 
+    # 'make gen_tar' creates this tarball path
+    kselftest_tarball = 'kselftest-packages/kselftest.tar.gz'
```

Comment:
Commit says ".xz", code expects ".gz", but depends on the default behavior of "gen_tar" without a FORMAT= specified. Perhaps explicitly declare "FORMAT=xz" (or =gz) (and update the filename and commit message appropriately)?

Reply:
Right, I'll make it more explicit and clean up the changelog.  Thanks for the review.

FWIW, I started with .xz for size reasons, but we're recycling a test-plan from another test project (Linaro LKFT) and their test wrappers etc. have a bunch of hard-coded assumptions for .gz, so I switched to that (but forgot to update commit message.)